client/blocks/edit-gravatar/index.jsx

```diff
@@ -0,0 +1,103 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import classnames from 'classnames';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import FilePicker from 'components/file-picker';
+import FormLabel from 'components/forms/form-label';
+import {
+	getCurrentUser,
+	isCurrentUserUploadingGravatar
+} from 'state/current-user/selectors';
+import Gravatar from 'components/gravatar';
+import { isOffline } from 'state/application/selectors';
+import { localize } from 'i18n-calypso';
+import Spinner from 'components/spinner';
+import * as OAuthToken from 'lib/oauth-token';
+import { uploadGravatar } from 'state/current-user/actions';
+
+export class EditGravatar extends Component {
+	constructor() {
+		super();
+		this.handleOnPick = this.handleOnPick.bind( this );
+	}
+
+	static propTypes = {
+		isOffline: PropTypes.bool,
+		translate: PropTypes.func,
+		user: PropTypes.object,
+		dispatch: PropTypes.func,
```

Comment:
Thanks! Fixed in b2a8380 ([related comment](https://github.com/Automattic/wp-calypso/pull/8456#discussion_r83313245))